Sources/BraintreePayPal/BTPayPalAccountNonce.m

```diff
@@ -16,6 +18,75 @@ @interface BTPayPalAccountNonce ()
 
 @implementation BTPayPalAccountNonce
 
+- (nullable instancetype)initWithJSON:(BTJSON *)json {
```

Comment:
I thought it made sense to have the JSON parsing contained to an initializer, instead of putting it somewhere in the `BraintreePayPalNative` module. At some point, we could update `BTPayPalDriver` to use this initializer too, instead of doing the parsing in the driver.

Reply:
+1 I like that idea of also updating BTPayPalDriver. I wouldn't be opposed to doing that in this PR. If not - making a jira or adding a TODO for it here. (I feel this is something we can easily forget)